docs/source/user-interfaces.rst

```diff
@@ -311,14 +311,14 @@ The approaches above should suffice to convert any interface into any other.
 We often see some anti-patterns that do not work as well:
 
 1.  Calling low-level APIs (delayed or futures) on high-level objects (like
-    Dask arrays or DataFrames). This downgrades those objects to their Numpy or
+    Dask arrays or DataFrames). This downgrades those objects to their NumPy or
     Pandas equivalents, which may not be desired.
     Often people are looking for APIs like ``dask.array.map_blocks`` or
-    ``dask.dataframe.map_partitions`` instead
+    ``dask.dataframe.map_partitions`` instead.
 2.  Calling ``compute()`` on Future objects.
-    Often people want the ``.result()`` method instead
-3.  Calling Numpy/Pandas functions on high-level Dask objects or
-    high-level Dask functions on Numpy/Pandas objects
+    Often people want the ``.result()`` method instead.
+3.  Calling NumPy/Pandas functions on high-level Dask objects or
+    high-level Dask functions on NumPy/Pandas objects
```

Comment:
Should we add a `.` at the end of this too?

Reply:
I'm not sure that this counts as a full sentence.  It may not stand on its own.

Reply:
There are a few things above that fall in the same category that either already had a `.` or had one added. Personally don't have strong feelings about this. Just something that caught my eye was all.